content/collections/experiment/en/experiment-quick-start.md

```diff
@@ -0,0 +1,266 @@
+---
+id: 1dc8fb12-66f4-4ff0-bf73-e2e52fcd78b1
+blueprint: experiment
+title: 'Experiment Quick Start'
+this_article_will_help_you:
+  - 'Setup the SDK and deployment for your experiments'
+  - 'Create feature flags, variants, and payloads'
+  - 'Design and implement the experiment'
+landing: false
+exclude_from_sitemap: false
+updated_by: 3f7c2286-b7da-4443-a04f-7c225af40254
+updated_at: 1753137993
+---
+Amplitude Experiment is a workflow-driven behavioral experimentation platform that accelerates the process of creating different variants of features and websites for experimentation.
+
+With Experiment, you can modify and configure product experiences for unique audiences through:
```

Comment:
Probably something we should align on, but for product names, when do we want to include "Amplitude," and when should we not?